### PR TITLE
Read default aws creds/keys from binded env vars

### PR DESCRIPTION
**Pull request by krzyzacy**

### CODE REVIEW

File: scenarios/kubernetes_aws.py [hunk 1]
```diff
@@ -208,6 +204,21 @@ def main(args):
...
     ARGS = PARSER.parse_args()
 
+    # If aws keys are missing, try to fetch from HOME dir
+    if not (ARGS.aws_ssh or ARGS.aws_pub or ARGS.aws_cred):
+        HOME = os.environ.get('HOME')
+        if not HOME:
+            raise ValueError('HOME dir not set!')
+        if not ARGS.aws_ssh:
+            ARGS.aws_ssh = '%s/.ssh/kube_aws_rsa' % HOME
+            print >>sys.stderr, 'AWS ssh key not found, try to fetch from %s' % ARGS.aws_ssh
```
Comment (zmerlynn) [inline scenarios/kubernetes_aws.py]: Nit: I think this should be "trying to fetch".

**Comment by zmerlynn** [comment]
Minor nit, feel free to self-LGTM after you fix it.

File: scenarios/kubernetes_aws.py (hunk 1 above)
Comment (zmerlynn) [inline scenarios/kubernetes_aws.py]: (And below)

Comment (krzyzacy) [inline scenarios/kubernetes_aws.py]: definitely need to took another grammar class :cry:

Comment (krzyzacy) [inline scenarios/kubernetes_aws.py]: ***need to take :cry: :cry:

Comment (fejta) [inline scenarios/kubernetes_aws.py]: I prefer the bikeshed reads Try actually, since progressive tense is weak :grin:

Comment (krzyzacy) [inline scenarios/kubernetes_aws.py]: @fejta you got it :-p

**Pull request merged**